vhdbuilder/packer/components.json

```diff
@@ -90,10 +90,11 @@
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
       "amd64OnlyVersions": [
-        "v1.4.9",
+        "v1.4.9"
+       ],
+      "multiArchVersions": [
         "v1.4.18"
```

Comment:
v1.4.18 should be in multiArchVersions so that it is cached on both amd64/arm64 vhd.